master/internal/db/postgres_snapshots_test.go

```diff
@@ -20,6 +20,8 @@ func TestCustomSearcherSnapshot(t *testing.T) {
 	etc.SetRootPath(RootFromDB)
 	db := MustResolveTestPostgres(t)
 	MustMigrateTestPostgres(t, db, MigrationsFromDB)
+	user := RequireMockUser(t, db)
+	exp := RequireMockExperiment(t, db, user)
```

Comment:
Using exp here because the database can get into a stale state (experiment_snapshots has a foreign key constraint of experiment in experiments table).